frontend/src/app/components/work-packages/work-package-comment/wp-comment-field.component.ts

```diff
@@ -32,14 +32,15 @@ import {
   FormattableEditFieldComponent,
   formattableFieldTemplate
 } from "core-app/modules/fields/edit/field-types/formattable-edit-field.component";
+import {InjectField} from "core-app/helpers/angular/inject-field.decorator";
 
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><del><span data-href="https://github.com/opf/openproject/blob/3b1eae5a64159f8a044579a25870c772186dbd3f/frontend/src/app/components/work-packages/work-package-comment/wp-comment-field.component.ts#L36"></span>Please use <code>ChangeDetectionStrategy.OnPush</code> for this component</del></td>
    </tr>
  </tbody>
</table>